common/game_concepts/wc_magic_lifestyles_concepts.txt

```diff
@@ -0,0 +1,68 @@
+﻿wc_magic_lifestyle = {
```

Comment:
AI seems to chooce the new lifestyles pretty consistantly now, however they should be likelier to choose Idolised magic over Accepted magic.

Reply:
(Light is accepted, Arcane is idolised)
![bild](https://user-images.githubusercontent.com/4825809/115442877-d2856280-a212-11eb-8a3f-f4f5aa153808.png)